<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>591</width>
    <height>568</height>
   </rect>
  </property>
  <property name="acceptDrops">
   <bool>true</bool>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>571</width>
      <height>131</height>
     </rect>
    </property>
    <property name="title">
     <string>选择：</string>
    </property>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>410</x>
       <y>30</y>
       <width>131</width>
       <height>91</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QPushButton" name="chose_file_button">
        <property name="text">
         <string>选择文件</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="chose_out_file_button">
        <property name="text">
         <string>文件路径</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>30</y>
       <width>321</width>
       <height>91</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLineEdit" name="input_line_text">
        <property name="dragEnabled">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="out_path_text"/>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>67</width>
       <height>71</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>文件路径：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>输出路径：</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>290</y>
      <width>571</width>
      <height>241</height>
     </rect>
    </property>
    <property name="title">
     <string>日志:</string>
    </property>
    <widget class="QPlainTextEdit" name="log_plainTextEdit">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>551</width>
       <height>201</height>
      </rect>
     </property>
     <property name="plainText">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>530</y>
      <width>571</width>
      <height>35</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="run_exec_button">
       <property name="text">
        <string>操  作</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="claer_log_button">
       <property name="text">
        <string>清空日志</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="export_log_button">
       <property name="text">
        <string>导出日志</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="about_me_button">
       <property name="text">
        <string>关  于</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QTabWidget" name="org_table_widget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>140</y>
      <width>571</width>
      <height>151</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string notr="true">常规选项</string>
     </attribute>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>0</y>
        <width>546</width>
        <height>111</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QRadioButton" name="decompile_apk_radio">
         <property name="text">
          <string>反编译APK</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QRadioButton" name="repack_apk_radio">
         <property name="text">
          <string>重打包apk</string>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QRadioButton" name="sign_apk_radio">
         <property name="text">
          <string>签名apk</string>
         </property>
        </widget>
       </item>
       <item row="0" column="3">
        <widget class="QRadioButton" name="repair_dex_radio">
         <property name="text">
          <string>修复dex</string>
         </property>
        </widget>
       </item>
       <item row="0" column="4">
        <widget class="QRadioButton" name="decompile_dex_radio">
         <property name="text">
          <string>反编译dex</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QRadioButton" name="repack_dex_radio">
         <property name="text">
          <string>重打包dex</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QRadioButton" name="dex_to_jar_radio">
         <property name="text">
          <string>dex转jar</string>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QRadioButton" name="jar_to_dex_radio">
         <property name="text">
          <string>jar转dex</string>
         </property>
        </widget>
       </item>
       <item row="1" column="3">
        <widget class="QRadioButton" name="open_jar_by_jui_radio">
         <property name="text">
          <string>jd-jui打开jar</string>
         </property>
        </widget>
       </item>
       <item row="1" column="4">
        <widget class="QRadioButton" name="open_jar_by_jadx_radio">
         <property name="text">
          <string>jadx打开jar</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QRadioButton" name="open_dex_by_jadx_radio">
         <property name="text">
          <string>jadx打开dex</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QRadioButton" name="open_apk_by_jadx_radio">
         <property name="text">
          <string>jadx打开apk</string>
         </property>
        </widget>
       </item>
       <item row="2" column="2">
        <widget class="QRadioButton" name="merge_dex_to_jar_radio">
         <property name="text">
          <string>多dex合并jar</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>其他选项</string>
     </attribute>
    </widget>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>chose_file_button</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>chose_file_()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>455</x>
     <y>83</y>
    </hint>
    <hint type="destinationlabel">
     <x>592</x>
     <y>183</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>chose_out_file_button</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>chose_dir()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>455</x>
     <y>124</y>
    </hint>
    <hint type="destinationlabel">
     <x>592</x>
     <y>138</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>run_exec_button</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>run_cmd()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>91</x>
     <y>568</y>
    </hint>
    <hint type="destinationlabel">
     <x>592</x>
     <y>517</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>claer_log_button</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>claer_log()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>199</x>
     <y>576</y>
    </hint>
    <hint type="destinationlabel">
     <x>588</x>
     <y>415</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>export_log_button</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>export_log()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>355</x>
     <y>575</y>
    </hint>
    <hint type="destinationlabel">
     <x>588</x>
     <y>362</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>show_about_me()</slot>
  <slot>chose_file_()</slot>
  <slot>chose_dir()</slot>
  <slot>run_cmd()</slot>
  <slot>claer_log()</slot>
  <slot>export_log()</slot>
 </slots>
</ui>
